UNRELEASED.md

```diff
@@ -84,6 +84,7 @@ Whitespace conventions:
 - Exceptions raised during the compilation now add to the backtrace the current location of the opal file if available (#1814).
 - Better use of `displayName` on functions and methods and more readable temp variable names (#1910)
 - Source-maps are now inlined and already contain sources, incredibly more stable and precise (#1856)
+- `Kernel#format` (and `sprintf` alias) are now in a dedicated module but still required by default (#1930)
```

Comment:
❤️

Reply:
I've updated the changelog to explain the breaking change